web/src/beta/features/Editor/hooks.ts

```diff
@@ -56,7 +57,12 @@ export default ({ tab }: { tab: Tab }) => {
     [setWidgetEditor],
   );
 
+  const { useGetLayersQuery } = useLayersFetcher();
+
+  const { nlsLayers = [] } = useGetLayersQuery({ sceneId, pollInterval: 500 });
```

Comment:
Why do you not do this in the useLayers.ts file? Seems appropriate to go in there, no?

Reply:
Done.